addons/binding/org.openhab.binding.geofence/src/main/java/org/openhab/binding/geofence/internal/BindingConstants.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.geofence.internal;
+
+
+import com.google.common.collect.Sets;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+
+import java.util.Set;
+
+/**
+ * Binding constants
+ *
+ * @author Gabor Bicskei - - Initial contribution
+ */
+public abstract class BindingConstants {
```

Comment:
Would be good to rename it to XYZBindingConstants as it is the case for all other bindings.